packages/xo-web/src/xo-app/vm/tab-advanced.js

```diff
@@ -422,6 +460,28 @@ export default class TabAdvanced extends Component {
                     </td>
                   </tr>
                 )}
+                {isVmRunning(vm) && (
+                  <tr>
+                    <th>{_('cpuMaskLabel')}</th>
+                    <td>
+                      <SingleLineRow>
+                        <Col size={4}>
+                          <XoSelect
+                            multi
+                            onChange={this._onChangeCpuMask}
+                            options={this._getCpuMaskOptions()}
+                            placeholder={_('selectCpuMask')}
+                            value={state.cpuMask}
```

Comment:
Shouldn't you provide a `renderer` here? What's the behaviour when you select more than one core?